make dev broken (gunicorn ssl error)

On master for me today:

```
seanh@mistakenot ~/P/h/h make dev
2015-06-03 12:18:51,646 [8573] [gunicorn.error:INFO] Starting gunicorn 19.3.0
2015-06-03 12:18:51,646 [8573] [gunicorn.error:INFO] Listening at: http://127.0.0.1:5000 (8573)
2015-06-03 12:18:51,647 [8573] [gunicorn.error:INFO] Using worker: h.server.Worker
2015-06-03 12:18:51,648 [8580] [gunicorn.error:INFO] Booting worker with pid: 8580
2015-06-03 12:18:51,655 [8580] [gunicorn.error:ERROR] Exception in worker process:
Traceback (most recent call last):
  File "/home/seanh/.virtualenvs/h/local/lib/python2.7/site-packages/gunicorn/arbiter.py", line 507, in spawn_worker
    worker.init_process()
  File "/home/seanh/.virtualenvs/h/local/lib/python2.7/site-packages/gunicorn/workers/ggevent.py", line 185, in init_process
    self.patch()
  File "/home/seanh/.virtualenvs/h/local/lib/python2.7/site-packages/gunicorn/workers/ggevent.py", line 65, in patch
    monkey.patch_all(subprocess=True)
  File "/home/seanh/.virtualenvs/h/local/lib/python2.7/site-packages/gevent/monkey.py", line 200, in patch_all
    patch_ssl()
  File "/home/seanh/.virtualenvs/h/local/lib/python2.7/site-packages/gevent/monkey.py", line 146, in patch_ssl
    patch_module('ssl')
  File "/home/seanh/.virtualenvs/h/local/lib/python2.7/site-packages/gevent/monkey.py", line 62, in patch_module
    gevent_module = getattr(__import__('gevent.' + name), name)
  File "/home/seanh/.virtualenvs/h/local/lib/python2.7/site-packages/gevent/ssl.py", line 5, in <module>
    from gevent import _sslgte279 as _source
  File "/home/seanh/.virtualenvs/h/local/lib/python2.7/site-packages/gevent/_sslgte279.py", line 46, in <module>
    orig_SSLContext = __ssl__.SSLContext
AttributeError: 'module' object has no attribute 'SSLContext'
Traceback (most recent call last):
  File "/home/seanh/.virtualenvs/h/local/lib/python2.7/site-packages/gunicorn/arbiter.py", line 507, in spawn_worker
    worker.init_process()
  File "/home/seanh/.virtualenvs/h/local/lib/python2.7/site-packages/gunicorn/workers/ggevent.py", line 185, in init_process
    self.patch()
  File "/home/seanh/.virtualenvs/h/local/lib/python2.7/site-packages/gunicorn/workers/ggevent.py", line 65, in patch
    monkey.patch_all(subprocess=True)
  File "/home/seanh/.virtualenvs/h/local/lib/python2.7/site-packages/gevent/monkey.py", line 200, in patch_all
    patch_ssl()
  File "/home/seanh/.virtualenvs/h/local/lib/python2.7/site-packages/gevent/monkey.py", line 146, in patch_ssl
    patch_module('ssl')
  File "/home/seanh/.virtualenvs/h/local/lib/python2.7/site-packages/gevent/monkey.py", line 62, in patch_module
    gevent_module = getattr(__import__('gevent.' + name), name)
  File "/home/seanh/.virtualenvs/h/local/lib/python2.7/site-packages/gevent/ssl.py", line 5, in <module>
    from gevent import _sslgte279 as _source
  File "/home/seanh/.virtualenvs/h/local/lib/python2.7/site-packages/gevent/_sslgte279.py", line 46, in <module>
    orig_SSLContext = __ssl__.SSLContext
AttributeError: 'module' object has no attribute 'SSLContext'
2015-06-03 12:18:51,656 [8580] [gunicorn.error:INFO] Worker exiting (pid: 8580)
Traceback (most recent call last):
  File "/home/seanh/.virtualenvs/h/bin/gunicorn", line 11, in <module>
    sys.exit(run())
  File "/home/seanh/.virtualenvs/h/local/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 74, in run
    WSGIApplication("%(prog)s [OPTIONS] [APP_MODULE]").run()
  File "/home/seanh/.virtualenvs/h/local/lib/python2.7/site-packages/gunicorn/app/base.py", line 189, in run
    super(Application, self).run()
  File "/home/seanh/.virtualenvs/h/local/lib/python2.7/site-packages/gunicorn/app/base.py", line 72, in run
    Arbiter(self).run()
  File "/home/seanh/.virtualenvs/h/local/lib/python2.7/site-packages/gunicorn/arbiter.py", line 174, in run
    self.manage_workers()
  File "/home/seanh/.virtualenvs/h/local/lib/python2.7/site-packages/gunicorn/arbiter.py", line 477, in manage_workers
    self.spawn_workers()
  File "/home/seanh/.virtualenvs/h/local/lib/python2.7/site-packages/gunicorn/arbiter.py", line 541, in spawn_workers
    time.sleep(0.1 * random.random())
  File "/home/seanh/.virtualenvs/h/local/lib/python2.7/site-packages/gunicorn/arbiter.py", line 214, in handle_chld
    self.reap_workers()
  File "/home/seanh/.virtualenvs/h/local/lib/python2.7/site-packages/gunicorn/arbiter.py", line 459, in reap_workers
    raise HaltServer(reason, self.WORKER_BOOT_ERROR)
gunicorn.errors.HaltServer: <HaltServer 'Worker failed to boot.' 3>
make: *** [dev] Error 1
```

Seems to be this commit <https://github.com/hypothesis/h/commit/c3c99186549e6ecdae54e6743aa90b9baae78cca>

--- a/gunicorn.conf.py
+++ b/gunicorn.conf.py
@@ -15,7 +15,7 @@ def post_fork(_server, _worker):
     # Support back-ported SSL changes on Debian / Ubuntu
     import _ssl
     import gevent.hub
-    if not hasattr(_ssl, '_sslwrap'):
+    if hasattr(_ssl, 'SSLContext') and not hasattr(_ssl, '_sslwrap'):
         gevent.hub.PYGTE279 = True
 
     try:

